packages/react/src/Card/Card.tsx

```diff
@@ -22,21 +22,25 @@ export type CardProps = {
   /**
    * Valid children include Card.Image, Card.Heading, and Card.Description
    */
-  children: React.ReactNode | React.ReactElement<CardHeadingProps> | React.ReactElement<CardDescriptionProps>
+  children:
+    | React.ReactNode
+    | React.ReactElement<CardLabelProps>
+    | React.ReactElement<CardHeadingProps>
+    | React.ReactElement<CardDescriptionProps>
   /**
    * The href of the link
    * */
   href: string
   /**
-   * Changes the link label of the card
+   * Changes the cta text of the card
    * * @default 'Learn more'
```

Comment:
I don't think we need the jsdoc annotations for prop names. they create noise and an expectation that we should use them more widely for prop descriptions. what do you think?